clients/drcachesim/tools/opcode_mix.cpp

```diff
@@ -347,34 +293,36 @@ opcode_mix_t::get_category_names(uint category)
 bool
 opcode_mix_t::print_results()
 {
-    shard_data_t total(0);
+    shard_data_t aggregated;
+    shard_data_t *total = &aggregated;
```

Comment:
Comment that we can't copy shard_data_t b/c of the unique ptr: that's the reason for this change, right?

Reply:
Yes, done.